src/EduardTrayan/CommonBundle/Controller/AbstractController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace EduardTrayan\CommonBundle\Controller\AbstractController;
+
+/**
+ * Class CommonController
+ * @package EduardTrayan\CommonBundle\Controller
+ */
+
+abstract class AbstractController
```

Comment:
Please register this controller as servcice
http://symfony.com/doc/current/cookbook/controller/service.html